src/java.base/windows/classes/sun/nio/fs/WindowsFileAttributes.java

```diff
@@ -412,6 +412,13 @@ boolean isDirectoryLink() {
         return isSymbolicLink() && ((fileAttrs & FILE_ATTRIBUTE_DIRECTORY) != 0);
     }
 
+    boolean isDirectoryJunction() {
+        if (isSymbolicLink())
+            return false;
+        return ((fileAttrs & FILE_ATTRIBUTE_DIRECTORY) != 0 &&
+                (fileAttrs & FILE_ATTRIBUTE_REPARSE_POINT) != 0);
```

Comment:
Should this just check if reparseTag has the value IO_REPARSE_TAG_MOUNT_POINT?

Reply:
So changed in 481bdbc.

Reply:
Thanks for these updates, the src changes look good now. I haven't looked at the test updates yet, will try to get to this soon.